### PR TITLE
Fix nits in the doc

### PR DESCRIPTION
**Pull request by waynesun09**
Spend few hours reading the doc and found some nits related to
typo, format, ref, etc., this fix did not include update on the
setup and testing.

Signed-off-by: Wayne Sun <gsun@redhat.com>

### CODE REVIEW

**Comment by will-Do** [comment]
ACK

File: docs/source/RegressionTestFarm.rst [hunk 1]
```diff
@@ -126,9 +126,9 @@ Then I'd create each machine with the appropriate labels
     Added host:
         foo-amd.bazcorp.com
 
-    $ /usr/local/autotest/cli/autotest-rpc-client host create -t amd64 -b hostprovisioning foo-intel.bazcorp.com
+    $ /usr/local/autotest/cli/autotest-rpc-client host create -t intel64 -b hostprovisioning foo-intel.bazcorp.com
```
Comment (xutian) [inline docs/source/RegressionTestFarm.rst]: I'm not sure, does the 'amd64' mean host arch is amd64, if so I think we need to keep this.

Thanks,
Xu

Comment (waynesun09) [inline docs/source/RegressionTestFarm.rst]: @xutian  this line is add intel64 arch with foo-intel.bazcorp.com host, I did not delete amd64 adding host command line.

**Comment by xutian** [comment]
thanks @waynesun09 .merged

**Pull request merged**